From e68382beae00a5a87c5011fdc41f322ab5e47c37 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Fri, 21 Sep 2018 00:50:16 +0200
Subject: [PATCH 15/17] Revert "soong_config: Add
 TARGET_NEEDS_PLATFORM_TEXT_RELOCATIONS"

* The per-process API level override will take care of this

This reverts commit cf7f7d48d0297f9a64413076d755b00e10e48c9a.

Change-Id: I3c0b5879ddf8e9dcc0b56a6e9a2855e515685389
---
 build/soong/android/variable.go | 4 ----
 build/soong/soong_config.mk     | 1 -
 2 files changed, 5 deletions(-)

diff --git a/build/soong/android/variable.go b/build/soong/android/variable.go
index 5f5c8501..746be3e7 100644
--- a/build/soong/android/variable.go
+++ b/build/soong/android/variable.go
@@ -20,9 +20,6 @@ type Product_variables struct {
 	Has_legacy_camera_hal1 struct {
 		Cflags []string
 	}
-	Needs_text_relocations struct {
-		Cppflags []string
-	}
 	Target_process_sdk_version_override struct {
 		Cppflags []string
 	}
@@ -52,7 +49,6 @@ type ProductVariables struct {
 	Device_support_wait_for_qsee  *bool `json:",omitempty"`
 	Has_legacy_camera_hal1  *bool `json:",omitempty"`
 	Java_Source_Overlays *string `json:",omitempty"`
-	Needs_text_relocations  *bool `json:",omitempty"`
 	Specific_camera_parameter_library  *string `json:",omitempty"`
 	Target_process_sdk_version_override *string `json:",omitempty"`
 	Target_shim_libs  *string `json:",omitempty"`
diff --git a/build/soong/soong_config.mk b/build/soong/soong_config.mk
index 09669711..9dbdece0 100644
--- a/build/soong/soong_config.mk
+++ b/build/soong/soong_config.mk
@@ -10,7 +10,6 @@ $(call add_json_bool, Device_support_legacy_hwfde, $(filter true,$(TARGET_LEGACY
 $(call add_json_bool, Device_support_wait_for_qsee, $(filter true,$(TARGET_KEYMASTER_WAIT_FOR_QSEE)))
 $(call add_json_bool, Has_legacy_camera_hal1, $(filter true,$(TARGET_HAS_LEGACY_CAMERA_HAL1)))
 $(call add_json_str, Java_Source_Overlays, $(JAVA_SOURCE_OVERLAYS))
-$(call add_json_bool, Needs_text_relocations, $(filter true,$(TARGET_NEEDS_PLATFORM_TEXT_RELOCATIONS)))
 $(call add_json_str, Specific_camera_parameter_library, $(TARGET_SPECIFIC_CAMERA_PARAMETER_LIBRARY))
 $(call add_json_str_omitempty, Target_process_sdk_version_override, $(TARGET_PROCESS_SDK_VERSION_OVERRIDE))
 $(call add_json_str_omitempty, Target_specific_headers_include_dir, $(TARGET_SPECIFIC_HEADER_PATH))
-- 
2.17.1

